CONTRIBUTING.md

```diff
@@ -23,7 +23,7 @@ Opening an issue is as easy as following [this link][issues] and filling out the
 - What did you try, step by step?
 - What did you expect?
 - What did happen instead?
-- What is the version of the Cozy Nef Konnector?
+- What is the version of the Cozy <APP_NAME>?
```

Comment:
Don't know if it's on purpose, but you seem to have left some template patterns.